src/model.rs

```diff
@@ -1484,16 +1484,16 @@ mod tests {
         // - stop_point_idx (usize -> index of one of the four test stop points)
         // - arrival_time (Time)
         // - departure_time (Time)
-        type VJConfig = (String, usize, Time, Time);
+        type VjConfig = (String, usize, Time, Time);
```

Comment:
Same, not sure, as `vj` (or `Vj`) is not a word.

Reply:
And then I read this: https://users.rust-lang.org/t/naming-convention-suggests-name-that-is-read-incorrectly/55002/27
So... Fine by me :shrug: , I'm OK to not argue against Rust convention on that one.

Reply:
Yes, `clippy` actually links to [this](https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case) and they're pretty specific about the acronyms...